packages/ra-core/src/controller/list/useListController.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import fakeDataProvider from 'ra-data-fakerest';
+
+import { CoreAdminContext } from '../../core';
+import { ListController } from './ListController';
+import type { DataProvider } from '../../types';
+import type { ListControllerResult } from './useListController';
+
+export default {
+    title: 'ra-core/controller/list/useListController',
+};
+
+const defaultDataProvider = fakeDataProvider(
+    {
+        posts: [
+            { id: 1, title: 'Post #1', votes: 90 },
+            { id: 2, title: 'Post #2', votes: 20 },
+            { id: 3, title: 'Post #3', votes: 30 },
+            { id: 4, title: 'Post #4', votes: 40 },
+            { id: 5, title: 'Post #5', votes: 50 },
+            { id: 6, title: 'Post #6', votes: 60 },
+            { id: 7, title: 'Post #7', votes: 70 },
```

Comment:
votes are unused. Also, can you use plausible titles using any Lorem Ipsum generator instead of these indexed titles?